Example/AsyncReactorExample/Features/Repository/Search/RepositorySearchView.swift

```diff
@@ -19,10 +19,11 @@ struct RepositorySearchView: View {
     @ActionBinding(RepositorySearchReactor.self, keyPath: \.query, action: RepositorySearchReactor.Action.enterQuery)
```

Comment:
For the throttle to actually work you need to add a cancelId here with type inFlight. Then as you type in the searchbar the old action will be cancelled if one is already running.
```
@ActionBinding(RepositorySearchReactor.self, keyPath: \.query, cancelId: .init(id: "enterQuery", mode: .inFlight), action: RepositorySearchReactor.Action.enterQuery)
    private var query: String
```